actions/setup_e2e_tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Any reason why we can't use the existing `st2_e2e_tests` workflow which performs those steps?

Just curious.